cmd/tidb-lightning-ctl/main.go

```diff
@@ -71,11 +71,13 @@ func run() error {
 		fsUsage = fs.Usage
 	}))
 
+	ctx := context.Background()
```

Comment:
Can we cancel the context by receiving a signal?

Reply:
Seems not currently. Because all the actions lightning-ctl provide should be processed quite fast, there is no need to cancel it.

Reply:
> Can we cancel the context by receiving a signal?

default action to usual signal could stop `tidb-lightning-ctl`, not graceful but I think that's OK.